keystone-init/README.md

```diff
@@ -174,6 +182,38 @@ containers:
 {{ include "keystone_env" .Values.my_pod.auth | indent(6) }}
 ```
 
+Services are created by their unique name:
+```yaml
+services:
+  example:
+    type:
+    region:
+    url:
+    description:
+    endpoints: []
+```
+In this case service with `example` name will be created first and then
+endpoint will be added to it with provided endpoints.
+`description` is optional.
+
+The `endpoints` block accepts a list of objects. As an example:
+```yaml
+services:
+  example:
+    # ...
+    url: http://top-url:8070/v2.0
+    endpoints:
+      - name: internal
```

Comment:
In overall, there are 3 possible endpoints **admin**, **internal**, **public**. So having a ```name``` as a word to depict what kind of endpoint we want to create is a bit misleading. I'd roll with ```type``` and maybe add some validation inside the python code to disallow other strings.

Reply:
Now code follow Keystone logic and naming convention.

Reply:
Peachy ;-)